mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -527,6 +528,10 @@ def match_log(self, line):
         if self.compare_log_idx < len(self.compare_log):
             regex = self.compare_log[self.compare_log_idx]
             # Either the line is matched as is or it is checked as a regular expression.
-            if regex in line or re.search(regex, line):
-                self.compare_log_idx += 1
+            try:
+                if regex in line or re.search(regex, line):
+                    self.compare_log_idx += 1
+            except error:
```

Comment:
You're totally right, my bad! And I learned something new today :smile: But man yeah, what a blissfully ambiguous error name, good going regex library lol.